### ISSUE TITLE
A way to cancel editting of a value

### ISSUE THREAD

**Issue by k1Tar1** [still-scene/t3#52]
Similar to blender, when you're changing the value in a node by sliding it you can press escape to cancel it. It could be also useful to cancel any action in general like moving nodes.

**Comment by pixtur** [comment]
Most actions (like editing input fields) should cancel when pressing the ESC key. Do you have a specific interaction that should have this behavior too?

**Comment by k1Tar1** [comment]
Firstly, I'm still using v3.2.0 and if what I'm gonna mention is already been dealt with in v3.3.0 I'm sorry for wasting your time. for editing a value like the value of a parameter for example yes you can use escape key to cancel your editing but only if you're typing the value but when using the wheel to change a value you can't cancel it and same goes for moving nodes you can't cancel it so the node goes back to its original place.

**Comment by pixtur** [comment]
Ah. I think I understand what you're after. Not just canceling but also reverting the changes in progress (e.g. undo the value editting when using the jog wheel)...

**Comment by k1Tar1** [comment]
Yeah, you got what I meant. Sorry if I wasn't clear before before.

**Comment by pixtur** [comment]
Verify if this issue is still a problem.

**Comment by pixtur** [comment]
This was implemented with v4.1
